src/js/factory/new-task-item.factory.js

```diff
@@ -0,0 +1,61 @@
+import { todoEl } from "../utils/dom-elements";
+
+class NewTaskItemFactory {
```

Comment:
A factory class that handles new tasks that should be inserted into the to-do list.